Makefile

```diff
@@ -50,7 +50,8 @@ K8S_APP_TAG := $(GRAVITY_TAG)
 TELEKUBE_APP_TAG := $(GRAVITY_TAG)
 WORMHOLE_APP_TAG := $(GRAVITY_TAG)
 LOGGING_APP_TAG ?= 5.0.3
-MONITORING_APP_TAG ?= 5.5.18
+MONITORING_APP_TAG ?= 5.5.18-5-g00d88d5c
+MONITORING_APP_BRANCH := sergei/grafana_cve
```

Comment:
Is it the correct branch name `sergei/grafana_cve` ?

Reply:
As usual, I do that just for PR testing. Whet it passes I am going to merge PR in monitoring, tag it and update Makefile.